tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1814,6 +1806,66 @@ Status StorageManager::load_array_schema(
   return st;
```

Comment:
(wanted to comment at end of if() block, but git app not letting me...)
If, just above, had !st.ok() and went through if(!st.ok()) block performing
```
*array_schema = nullptr;
```
, then the line [#1805?] just above following that if() block is likely to fail
```
(*array_schema)->set_uri(schema_uri);
```
probably need to 'return st' from bottom of if() block... ?

Reply:
^^@bdeng-xt

Reply:
thanks@dhoke-4tdb